packages/studio-base/src/dataSources/RosbridgeDataSourceFactory.tsx

```diff
@@ -14,6 +14,9 @@ class RosbridgeDataSourceFactory implements IDataSourceFactory {
   type: IDataSourceFactory["type"] = "connection";
   displayName = "Rosbridge (ROS 1 & 2)";
   iconName: IDataSourceFactory["iconName"] = "Flow";
+  docsLink = "/rosbridge";
+  docsIntro =
+    "Connect to a running ROS 1 or ROS 2 system over WebSockets to access your ROS nodes directly.";
```

Comment:
```suggestion
    "Connect to a running ROS 1 or ROS 2 system over WebSockets using the rosbridge_suite package.";
```
Not married to this wording suggestion, but I think it is helpful to clarify that this connection is meant for use with an off-the-shelf server package.